NewEvaluation.html

```diff
@@ -82,25 +88,26 @@
 
 		<div class="user-textarea">
 			<h3>Should the candidate be hired?</h3>
-			<textarea  style= "width:1200px ; height: 200px;" placeholder="The type of project that is suitable for the candidate &#13;&#10;Is guidance requiredd for the candidate"></textarea>
+			<textarea placeholder="The type of project that is suitable for the candidate &#13;&#10;Is guidance requiredd for the candidate"></textarea>
```

Comment:
Q2:required is that before it does any action it requires user input to use put in the text box.
Q1: those are the specific values of the ascii code for HTML. &#10 is for input of new line. and &#13 for place the cursor at the start of the line